### PR TITLE
pythonPackages.rdflib-jsonld: init at 0.5.0

### PR DESCRIPTION
**Pull request by koslambrou**
#### Motivation for this change
added rdflib-jsonld python package

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  name = "rdflib-jsonld";
+  version = "0.5.0";
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/f5/17/45e137be0d93b70827fe5529c0400731344a978bc792193d7d9152e6dbe4/${name}-${version}.tar.gz";
+    sha256 = "4f7d55326405071c7bce9acf5484643bcb984eadb84a6503053367da207105ed";
+};
+  doCheck = true;
+  format = "setuptools";
+  propagatedBuildInputs = [ rdflib ];
+  meta = with lib; {
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:17]: format please
```suggestion
{ buildPythonPackage
, fetchurl
, lib

, rdflib
}:

buildPythonPackage rec {
  pname = "rdflib-jsonld";
  version = "0.5.0";
  
  src = fetchurl {
    url = "https://files.pythonhosted.org/packages/f5/17/45e137be0d93b70827fe5529c0400731344a978bc792193d7d9152e6dbe4/${name}-${version}.tar.gz";
    sha256 = "4f7d55326405071c7bce9acf5484643bcb984eadb84a6503053367da207105ed";
};

  doCheck = true;

  propagatedBuildInputs = [ rdflib ];
  
  meta = with lib; {
```

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage
+, fetchurl
+, lib
+
+, rdflib
+}:
+buildPythonPackage rec {
+  name = "rdflib-jsonld";
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:8]:
```suggestion
  pname = "rdflib-jsonld";
```

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  version = "0.5.0";
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/f5/17/45e137be0d93b70827fe5529c0400731344a978bc792193d7d9152e6dbe4/${name}-${version}.tar.gz";
+    sha256 = "4f7d55326405071c7bce9acf5484643bcb984eadb84a6503053367da207105ed";
+};
+  doCheck = true;
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:14]: default
```suggestion

```

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  format = "setuptools";
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:15]: default
```suggestion

```

File: maintainers/maintainer-list.nix [hunk 5]
```diff
@@ -4414,6 +4414,12 @@
     githubId = 524268;
     name = "Koral";
   };
+  koslambrou = {
+    email = "koslambrou@gmail.com";
+    github = "koslambrou";
+    githubId = 2037002;
+    name = "Konstantinos";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix:4422]: Please make this into a separate commit:

```
maintainers: add <name>
```

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+, rdflib
+}:
+buildPythonPackage rec {
+  name = "rdflib-jsonld";
+  version = "0.5.0";
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/f5/17/45e137be0d93b70827fe5529c0400731344a978bc792193d7d9152e6dbe4/${name}-${version}.tar.gz";
+    sha256 = "4f7d55326405071c7bce9acf5484643bcb984eadb84a6503053367da207105ed";
+};
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:13]: use fetchPypi
```suggestion
  src = fetchPypi {
   inherit pname version;
    sha256 = ...;
};
```

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  doCheck = true;
+  format = "setuptools";
+  propagatedBuildInputs = [ rdflib ];
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:16]:
```suggestion
  nativeBuildInputs = [ nose ];

  propagatedBuildInputs = [ rdflib ];
```

```
  error: Command '['/nix/store/fjgnz0xfl04hsblsi4ym5y5akfh6mlmy-python3-3.8.5/bin/python3.8', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/build/tmpf7v5k5qr', '--quiet', 'nose']
```

**Comment by jonringer** [comment]
@koslambrou thanks for opening your first PR

please look at https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/rope/default.nix for an example of a python package which uses nose

File: maintainers/maintainer-list.nix (hunk 5 above)
Comment (koslambrou) [reply maintainers/maintainer-list.nix:4422]: a commit in this PR or in a new PR ?

**Comment by jonringer** [comment]
i notcied you force pushed after closing the PR, did you mean to do that?

**Comment by koslambrou** [comment]
Oups not at all. Just finished creating the 2 commits with all the suggestions in my branch.

File: pkgs/development/python-modules/rdflib-jsonld/default.nix (hunk 7 above)
Comment (koslambrou) [reply pkgs/development/python-modules/rdflib-jsonld/default.nix:11]: What command told you that `nose` was missing in my dependencies ?

**Review by jonringer — APPROVED** [review]
commits LGTM
has tests :+1: 

```
https://github.com/NixOS/nixpkgs/pull/95711
3 packages built:
python27Packages.rdflib-jsonld python37Packages.rdflib-jsonld python38Packages.rdflib-jsonld
```

sorry, another review below. Very close

File: pkgs/development/python-modules/rdflib-jsonld/default.nix (hunk 7 above)
Comment (jonringer) [reply pkgs/development/python-modules/rdflib-jsonld/default.nix:11]: `'--no-deps', '-w', '/build/tmpf7v5k5qr', '--quiet', 'nose']` it's at the end

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 8]
```diff
@@ -0,0 +1,18 @@
+{ buildPythonPackage, fetchPypi, lib, rdflib, nose }:
+
+buildPythonPackage rec {
+  pname = "rdflib-jsonld";
+  version = "0.5.0";
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:5]: for readability
```suggestion
  version = "0.5.0";

```

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4f7d55326405071c7bce9acf5484643bcb984eadb84a6503053367da207105ed";
+  };
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:9]:
```suggestion
  };
```

File: pkgs/development/python-modules/rdflib-jsonld/default.nix [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+  nativeBuildInputs = [ nose ];
+  propagatedBuildInputs = [ rdflib ];
```
Comment (jonringer) [inline pkgs/development/python-modules/rdflib-jsonld/default.nix:11]:
```suggestion
  propagatedBuildInputs = [ rdflib ];

```

**Review by jonringer — APPROVED** [review]
otherwise LGTM

```
https://github.com/NixOS/nixpkgs/pull/95711
3 packages built:
python27Packages.rdflib-jsonld python37Packages.rdflib-jsonld python38Packages.rdflib-jsonld
```

**Comment by koslambrou** [comment]
@jonringer I finished applying the changes. Thanks for the review :)

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM
has tests :+1: 

```
https://github.com/NixOS/nixpkgs/pull/95711
3 packages built:
python27Packages.rdflib-jsonld python37Packages.rdflib-jsonld python38Packages.rdflib-jsonld
```
